be/src/olap/rowset/segment_v2/segment_writer.cpp

```diff
@@ -471,7 +471,7 @@ Status SegmentWriter::append_block_with_partial_content(const vectorized::Block*
             }
 
             if (!_opts.rowset_ctx->partial_update_info->can_insert_new_rows_in_partial_update) {
-                return Status::InternalError(
+                return Status::Error<INVALID_SCHEMA, false>(
```

Comment:
We may attach some context to the msg: a transaction id or load id

Reply:
should attach the load_id at _flush_memtable